src/pages/Token/Index.tsx

```diff
@@ -0,0 +1,68 @@
+import {useParams} from "react-router-dom";
+import {gql, useQuery} from "@apollo/client";
+import {Stack, Grid} from "@mui/material";
+import React from "react";
+import HeaderSearch from "../layout/Search/Index";
+import TokenTitle from "./Title";
+import TokenTabs from "./Tabs";
+
+const TOKEN_QUERY = gql`
+  {
+    token_datas(where: {name_hash: {}, name: {_eq: "Aptos Zero: 7"}}) {
```

Comment:
Remove `name: {_eq: "Aptos Zero: 7"}`?

Reply:
We'll be querying the token_datas by token_id. Bowen is still working on adding the token_id so I query it by name as a temporary solution. The idea is to have some data to fill the token page. I'll change it once we have the token_id ready.